src/test/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollectionTest.java

```diff
@@ -102,13 +109,44 @@ public void testNoIntersection(IntervalArgumentCollection iac){
         iac.addToIntervalStrings("1:10-20");
         iac.addToIntervalStrings("1:50-200");
         iac.intervalSetRule = IntervalSetRule.INTERSECTION;
-        iac.getIntervals(hg19GenomeLocParser);
+        iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary());
```

Comment:
Reverted, as it would require an `@BeforeClass` due to the way `hg19GenomeLocParser` is initialized